var/spack/repos/builtin/packages/py-onnx-runtime/package.py

```diff
@@ -126,4 +127,9 @@ def cmake_args(self):
 
     @run_after("install")
     def install_python(self):
-        PythonPackage.install(self, self.spec, self.prefix)
+        platlib = self.spec["python"].package.platlib
+        with working_dir(self.build_directory):
+            py = which("python")
+            py("setup.py", "build")
```

Comment:
Direct invocation of `setup.py` is deprecated, see https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html and #27798. We should instead use a build tool like pip.